runtime/jvmti/jvmtiHook.c

```diff
@@ -145,6 +145,10 @@ static void jvmtiHookGetEnv (J9HookInterface** hook, UDATA eventNum, void* event
 static void jvmtiHookVmDumpStart (J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData);
 static void jvmtiHookVmDumpEnd (J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData);
 static UDATA methodExists(J9Class * methodClass, U_8 * nameData, UDATA nameLength, J9UTF8 * signature);
+#if JAVA_SPEC_VERSION >= 19
+static void jvmtiHookVirtualThreadStarted (J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData);
+static void jvmtiHookVirtualThreadEnd (J9HookInterface** hook, UDATA eventNum, void* eventData, void* userData);
```

Comment:
```suggestion
static void jvmtiHookVirtualThreadStarted (J9HookInterface **hook, UDATA eventNum, void *eventData, void *userData);
static void jvmtiHookVirtualThreadEnd (J9HookInterface **hook, UDATA eventNum, void *eventData, void *userData);
```

Reply:
I am leaning towards keeping the existing format here, or else we could just change all the function headers to match standard